requirements/nightly.in

```diff
@@ -3,4 +3,4 @@
 # --- END OF CUSTOM SECTION ---
 # The following was generated by 'tox -e deps', DO NOT EDIT MANUALLY!
 matplotlib>=3.5
-https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
+https://github.com/scipp/scipp/releases/download/nightly/scipp-nightly-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
```

Comment:
I don't think we have dropped 3.10 yet?

Reply:
Not sure why this changed. I just ran `tox -e deps` :confused:

Reply:
Yeah, it picks up the local env python and forces it through https://github.com/scipp/copier_template/issues/163

Reply:
Good to know, thanks!